packages/footer/src/js/react.js

```diff
@@ -23,17 +23,19 @@ import PropTypes from 'prop-types';
  *
  * @param  {node}   children         - The inside of this section
  * @param  {string} className        - An additional class, optional
+ * @param  {string} ariaLabel        - The aria-label attribute, optional
  * @param  {object} attributeOptions - Any other attribute options
  */
-export const AUfooterNav = ({ children, className = '', ...attributeOptions }) => (
-	<nav className={`au-footer__navigation ${ className }`} { ...attributeOptions }>
+export const AUfooterNav = ({ children, className = '', ariaLabel = 'footer navigation', ...attributeOptions }) => (
```

Comment:
Have a look at `defaultProps` as I think we should probably be using those instead of this approach for consistency.